all-together.ipynb

```diff
@@ -0,0 +1,775 @@
+{
```

Comment:
<p>Here you assume that the user has a folder <code>datasets</code>  with tranX exactly one level above our repository. It would be better to keep all the work in one folder and clone tranX in our folder if it's missing (just add a helper function that checks whether the folder exists and clone if it does not).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/JetBrains-Research/metrics-evaluation/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>